common/service/config/config.go

```diff
@@ -219,6 +222,8 @@ type (
 	}
 
 	// ClustersInfo contains the all cluster names and active cluster
+	//
+	// Deprecated: please use ClusterInformation instead
 	ClustersInfo struct {
```

Comment:
I would recommend to remove all configs in ClustersInfo(make it an empty struct) and keep our code simple, and panic if ClustersInfo is specified.